src/Content/Assets.php

```diff
@@ -74,13 +90,16 @@ protected function getPreference(Request $request)
 
         if ($actor->getPreference('fofNightMode_perDevice')) {
             return (int) Arr::get($request->getCookieParams(), 'flarum_nightmode', $default);
-        } else {
-            return (int) $actor->getPreference('fofNightMode', $default);
         }
+
+        return (int) $actor->getPreference('fofNightMode', $default);
```

Comment:
Adjusted the else statement on line 77-78, as it is not needed as a preference will always be returned to the calling method.